src/pages/signup.js

```diff
@@ -11,12 +11,14 @@ import gear from "../assets/gear.svg"
 import PasswordStrengthMeter from "../components/passwordstrengthmeter"
 
 import { faGoogle, faGithub } from "@fortawesome/free-brands-svg-icons"
-import {auth,
+import {
```

Comment:
Codacy found an issue: [Unexpected line break after this opening brace.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=6000161)